TicketsDemo.Domain/DefaultImplementations/ReservationLogger.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using TicketsDemo.Domain.Interfaces;
+
+namespace TicketsDemo.Domain.DefaultImplementations
+{
+    public class ReservationLogger : ILogger
```

Comment:
this logger is not specific to reservation, it can be used by any business feature, please rename

Reply:
Now it is called `ReservationServiceLogging` which is again makes an impression that the logger is specific to reservation service, while it is not.

It can be used across various business feature. 
The name shall reflect it's meaning, and the meaning is that this logger is logging to file.